src/modules/static-pages/about/components/section/video/index.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from "react";
+import { Player, BigPlayButton } from 'video-react';
+import poster from 'assets/images/postar.jpeg'
+import 'video-react/dist/video-react.css'; 
+import video from "assets/video/anmy.mp4";
+
+ 
+export default class AboutVideo extends Component {
+  
+  render() {
+  
+     return (
+         <div>
+            <Player src={video} 
+                    poster={poster}          
+             >
+               <BigPlayButton position="center" />
```

Comment:
The button gets the focus but no sign for that, try to navigate through the `tab` button. Try to make the button more accessible.

Reply:
Also, try to make the text of video control bigger

Reply:
I don't know exactly what  I should do I'm not  familiar for this case, anyway I tried to fix it